conda-forge.yml

```
@@ -7,3 +7,6 @@ github:
 provider:
   linux_aarch64: default
   linux_ppc64le: default
+noarch_platforms:
+  - linux_64
+  - win_64
```

Swapping to keep them alphabetically sorted. Bit migration PRs like resorting these even if nothing else changes. So this will help avoid some no-op bot PRs hopefully

```suggestion
noarch_platforms:
  - linux_64
  - win_64
provider:
  linux_aarch64: default
  linux_ppc64le: default
```